client/jetpack-connect/plans.jsx

```diff
@@ -255,6 +258,7 @@ const connectComponent = connect(
 			selectedSite,
 			selectedSiteSlug,
 			userId: user ? user.ID : null,
+			countryCode: requestGeoLocation().data,
```

Comment:
Hmm, should we also add this to `<PlansLanding />` too? Because the ab test is working fine in `/jetpack/connect/plans` but not in `/jetpack/connect/store` for me - we never seem to request the user's geo location in `<PlansLanding />`.

Reply:
good catch!